315 feature add user approvals page

Aradon267 opened the pull request:
This PR adds the ability to activate and deactivate users from the website(only admins can do this, and admins can't be created or demoted).
This is how it looks:
![image](https://github.com/user-attachments/assets/d279c014-896f-4c31-981d-c51cc31d102d)

Currently, it's a long table with inactive users at the top and admins at the bottom, not sure if filtering is a must at the moment.
The username also redirects to their profile.

lordYorden commented on cv_next/app/admin/components/profilesTable.tsx:
```diff
@@ -0,0 +1,68 @@
+                  className=" hover:underline"
+                  href={`/profile/${profile.username}`}
+                >
+                  {profile.username}
```
add something if the user isn't set yet

lordYorden commented on cv_next/types/database.types.ts:
what are all these changes beside `profile_perms`

Aradon267 replied:
I don't know, this file was generated from supabase directly

Aradon267 commented:
Add approve all inactive

Aradon267 commented:
Added a button like this
![image](https://github.com/user-attachments/assets/7a2abeb7-8e08-425b-b006-a2369f5fba2b)

Also added text if no username exists
![image](https://github.com/user-attachments/assets/abfd3ddd-f910-4295-9b4a-05e894725688)

lordYorden commented:
will check later today looks good

lordYorden reviewed (approved):
Looks good
